src/app/home/home.component.ts

```diff
@@ -49,39 +52,53 @@ export class HomeComponent implements OnInit {
         this.introRes = res;
         this.intro = marked((this.langService.lang === 'english') ? res.introEn : res.introFa);
       });
-    this.contentfulService.getHomeData()
-      .then(res => {
-        let slideshows = res[0].fields.slideShow;
+    this.getJsonFileService.getTechnologyData()
+      .then((res) => {
+        console.log('res', res);
+        this.slideShows = res;
         let _technologies = res[1].fields.technologies;
+
```

Comment:
remove this variable : _technologies and all its associations, so its for last vaersion of code and you,ve defined your own variables, so no need to _technologies